hw/hw_0001/index.3.js

```diff
@@ -1,17 +1,26 @@
 let currencies = [
-    {name: "hrn", cost: 1},
-    {name: "usd", cost: 25.2},
-    {name: "eur", cost: 30.1},
+  {shortname: "AUD", name: "Австралійський долар", cost: 16.155},
+  {shortname: "AZD", name: "Азербайджанський манат", cost: 14.4655},
+  {shortname: "BYN", name: "Бiлоруський рубль", cost: 10.9976 },
+  {shortname: "HKD", name: "Гонконгівський долар", cost: 3.1511},
+  {shortname: "DKK", name: "Данська крона", cost: 3.6037},
+  {shortname: "USD", name: "Долар США", cost: 24.561},
+  {shortname: "EUR", name: "Євро", cost: 26.9299},
+  {shortname: "EGP", name: "Єгипетський фунт", cost: 1.5739},
+  {shortname: "PLN", name: "Злотий", cost: 6.2426},
+  {shortname: "CAD", name: "Канадський долар", cost: 18.3881},
+  {shortname: "HRK", name: "Куна", cost: 3.6095},
+  {shortname: "MDL", name: "Молдовський лей", cost: 1.3754},
```

Comment:
Please, add a separate position with currency HRN with exchange rate 1

Reply:
Well I added but used the official name **UAH** instead **HRN**